.github/workflows/lint.yml

```
@@ -37,27 +37,35 @@ jobs:
       - name: Run jsonlint
         run: |
           jsonlint packer/*json
-      - name: Fetch and unzip packer nightly
-        run: |
-          wget -nv -O $HOME/packer.zip https://github.com/hashicorp/packer/releases/download/nightly/packer_linux_amd64.zip
-          unzip -d $HOME/packer $HOME/packer.zip
+      - name: Install Hashicorp repo
+        run: >
+          wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor |
+          sudo tee /usr/share/keyrings/hashicorp-archive-keyring.gpg &&
+          echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg]
+          https://apt.releases.hashicorp.com $(lsb_release -cs) main" |
+          sudo tee /etc/apt/sources.list.d/hashicorp.list
```

Since we're already running as root, we can trim this and remove some of the calls to `tee` to more closely match the commands at https://wiki.debian.org/DebianRepository/UseThirdParty. Honestly... that page mentions that de-armoring is only required pre-`stretch`, so we may be able to just `wget` directly to `$KEYRING_FILE` but ehh.

```suggestion
        run: |
          KEYRING_FILE="/usr/share/keyrings/hashicorp-archive-keyring.gpg"
          curl -SL https://apt.releases.hashicorp.com/gpg | gpg --dearmor > "$KEYRING_FILE"
          echo "deb [signed-by=$KEYRING_FILE] https://apt.releases.hashicorp.com $(lsb_release -cs) main" > /etc/apt/sources.list.d/hashicorp.list
```

---

I'm not convinced we're actually root by default. This failed hard when I tried it.